lib/facilities/configuration.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'common/client/configuration/rest'
 require 'common/client/middleware/response/raise_error'
+require 'common/client/middleware/response/facility_validator'
 require 'common/client/middleware/response/facility_parser'
 require 'typhoeus'
```

Comment:
I know it's not the main thrust of this PR but the reason we brought in typhoeus was to do parallel queries to the ArcGIS services when we were querying on each map refresh. Now that that's no longer the request model, you could do away with typhoeus both here and in the Gemfile.

Reply:
Thanks for pointing this out, I'll definitely pull it from here. I need to dig a bit more but it looks like RX client might still use typhoeus.. but I'll yank the gem if possible.. killing off dependencies is one of my favorite things!!